components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts

```diff
@@ -79,4 +90,34 @@ export class ServiceGroupsRequests {
 
     return this.httpClient.get<HealthSummary>(url);
   }
+
+  public getSuggestions(
+    field_name: string,
+    query_fragment: string,
```

Comment:
For consistency should use camel case instead of snake case, so `fieldName` and `queryFragment`. (I think it is OK that they are not identical to the URL params.)